subprojects/docs/src/docs/release/notes.md

```diff
@@ -212,6 +212,19 @@ The default tool versions of the following code quality plugins have been update
   In addition, the default ruleset was changed from the now deprecated `java-basic` to `category/java/errorprone.xml`.
   We recommend configuring a ruleset explicitly, though.
 
+### Library upgrades
+
+Several libraries that are used by Gradle have been upgraded:
+
+- Ant has been upgraded from 1.9.11 to 1.9.13.
+- The AWS SDK used to access S3 backed Maven/Ivy repositories has been upgraded from 1.11.267 to 1.11.407.
+- The BND library used by the OSGi plugin has been upgraded from 3.4.0 to 4.0.0.
+- The Google Cloud Storage JSON API Client Library used to access Google Cloud Storage backed Maven/Ivy repositories has been upgraded from v1-rev116-1.23.0 to v1-rev136-1.25.0.
+- Ivy has been upgraded from 2.2.0 to 2.3.0.
+- The JUnit Platform JARs used by the `Test` have been upgraded from JUnit Platform 1.0.3 to 1.3.1.
```

Comment:
typo: "by the `Test` *task*"

Reply:
Also, you have "JUnit Platform" twice, I'd skip the second mention.

Reply:
Both fixed, thanks!